gradle-jdks-setup-common/build.gradle

```diff
@@ -14,3 +14,8 @@ dependencies {
 javaVersion {
     target = 11
 }
+
+// Checkstyle does not run with Java 11
```

Comment:
let's be a bit more specific here: version of checkstyle is compiled with java 17

Reply:
curious, does the `javaCompiler` explicitly specified here do anything?